### PR TITLE
minor tweaks to style and docs

### PR DESCRIPTION
**Pull request by maurolepore**
This PR adds small changes I noticed in a different PR but pushed separately to keep the diffs clear.

- Redocument 
- Style
- Spellcheck then render readme

### CODE REVIEW

File: R/plot_emission_intensity.R [hunk 1]
```diff
@@ -15,10 +15,12 @@
...
-  check_plot_emission_intensity(data, env = list(data = substitute(data)))
+  env <- list(data = substitute(data))
+  check_plot_emission_intensity(data, env = env)
 
-  prep <- prep_emission_intensity(data)
-  plot_emission_intensity_impl(prep)
+  data %>%
+    prep_emission_intensity() %>%
+    plot_emission_intensity_impl()
 }
```
Comment (maurolepore) [inline R/plot_emission_intensity.R:24]: The body of all `plot_*()` functions now has a consistent style:

- Define `env`
- Call `check_*()`
- White line
- `data %>% ...`

File: README.Rmd [hunk 2]
```diff
@@ -76,7 +76,7 @@ data <- market_share %>%
 qplot_techmix(data)
 ```
 
-* Use `plot_*()` for a more 'bare' plot that you can customize yourself by 
+* Use `plot_*()` for a more "bare" plot that you can customize yourself by
```
Comment (maurolepore) [inline README.Rmd:79]: For some reason `R CMD check` disliked the single quotes and highlighted `bare'` as a potential spelling error.

File: tests/testthat/test-plot_techmix.R [hunk 3]
```diff
@@ -185,28 +185,30 @@ test_that("When data has 'label_tech' it is used in the plot", {
...
+      sector == "power",
+      region == "global",
+      metric %in% c("projected", "corporate_economy", "target_sds")
+    ) %>%
+    mutate(metric = factor(
+      .data$metric,
+      levels = c("corporate_economy", "projected", "target_sds")
+    )) %>%
+    arrange(.data$metric) %>%
+    mutate(metric = as.character(.data$metric))
```
Comment (maurolepore) [inline tests/testthat/test-plot_techmix.R:211]: This is automatic work of the styler package.

**Review by MonikaFu — APPROVED** [review]
LGTM

**Pull request merged**